odl/trafos/backends/pyfftw_bindings.py

```diff
@@ -29,12 +29,18 @@
 try:
     import pyfftw
     PYFFTW_AVAILABLE = True
-    if parse_version(pyfftw.__version__) < parse_version('0.10.4'):
-        warnings.warn('PyFFTW < 0.10.4 is known to cause problems with some '
-                      'ODL functionality, see issue #1002.',
-                      RuntimeWarning)
 except ImportError:
     PYFFTW_AVAILABLE = False
+else:
+    import pip
```

Comment:
I don't like adding `pip` as a dependency to ODL, and further you mention this taking some performance.

I'd rather remove this, or as mentioned somewhere else explcitly check against something like

```python
if parse_version(pyfftw.__version__) < parse_version('0.10.3.devSOMETHING'):
    warnings.warn('PyFFTW < 0.10.4 is known to cause problems with some '
                  'ODL functionality, see issue #1002.',
                  RuntimeWarning)
```